prime-router/src/main/kotlin/azure/SettingsFacade.kt

```diff
@@ -38,6 +39,8 @@ class SettingsFacade(
         // Format OffsetDateTime as an ISO string
         mapper.registerModule(JavaTimeModule())
         mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+        // Unknown properties in the payload may be caused by new versions of the CLI of the code
+        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
```

Comment:
Perhaps we should add a warning if we attempt to deserialize something with unknown properties.  Seems like the schema/data structures could get a bit out of hand if we never backfill / repair older records.